keyboards/handwired/d48/d48.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "quantum.h"
+
+#define NOPE KC_NO
```

Comment:
Best to not introduce another convention here, most existing use cases are  `___`, `XXX`, `_x_`.
```suggestion
#define ___ KC_NO
```